docs/source/peers/peers.md

```diff
@@ -160,6 +167,22 @@ Later in this section, you'll learn more about the detailed nature of this
 ordering process --- and for a really detailed look at this process see the
 [Transaction Flow](../txflow.html) topic.
 
+### Using Fabric Gateway
+
+Hyperledger Fabric v2.4 introduces the optional Fabric Gateway to simplify client
+application development. Fabric Gateway (the gateway) migrates code for transaction
+proposals and endorsements to the gateway peer, which then interacts with the channels
```

Comment:
No need to mention 'channels', since these requests are in the scope of a single channel.

Reply:
ok